### PR TITLE
ElrondApes WL

### PR DESCRIPTION
**Pull request by therealmasterj**

### CODE REVIEW

File: LKMEX-whitelist/whitelist_request_template.md [hunk 1]
```diff
@@ -3,56 +3,63 @@
 ## Transferability
 
 To conserve its energy and protect its benefits, old and new LKMEX will no longer be transferable, subsequently bound to
-the account where it is currently held. Some limited transfers for LKMEX will continue to be allowed - check the 
+the account where it is currently held. Some limited transfers for LKMEX will continue to be allowed - check the
```
Comment (sasurobert) [inline LKMEX-whitelist/whitelist_request_template.md]: revert the changes on the template document.

File: LKMEX-whitelist/ElrondApes.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+Our dApp (https://app.elrondapes.com/) currently provides a dashboard collection metrics, a community map and a staking feature. Basically, by staking your ElrondApes NFT into our contract, you're receiving LKMEX token as weekly rewards. These rewards come from a percentage of the sales royalties and the farming on the pool EGLD/USDC. We're internally discussing about changing the reward token, since the LKMEX will become non transferrable soon. We need to be whitelisted since stakers have not claimed their rewards from the contract yet.
+
+## Address of Smart Contract
+
+https://explorer.elrond.com/accounts/erd1qqqqqqqqqqqqqpgq575mqgaqe4f53rqc5ssczkxtsx4fs26l3azq9zwkaw
+
+## Smart Contract Code
+
+Currently private repository
```
Comment (sasurobert) [inline LKMEX-whitelist/ElrondApes.md:21]: We need access to the code.

**Review by sasurobert — CHANGES_REQUESTED** [review]

File: LKMEX-whitelist/ElrondApes.md (hunk 2 above)
Comment (therealmasterj) [reply LKMEX-whitelist/ElrondApes.md:21]: I just added you to the repo. Tell me if I need to add other collaborators

Comment (therealmasterj) [reply LKMEX-whitelist/ElrondApes.md:21]: https://github.com/EapesClub/elrond-apes-core-sc/blob/main/contracts/staking/src/rewards.rs

**Review by sasurobert — APPROVED** [review]
Contract satisfies the requirements. We will follow up with the whitelisting. PR approved.

**Review by andrei-marinica — APPROVED** [review]
